### PR TITLE
XOR-281 [Feat] Ensure reorder functionality added to DS Security Rules

### PR DESCRIPTION
**Pull request by NikitaJadhavXoriant**
**Product areas affected**
Studio -> App Data -> Data Source -> Security Rules -> Reorder rules

**What does this PR do?**
Implemented reorder functionality in DS Security Rules

**JIRA ticket**
[JIRA](https://weboo.atlassian.net/browse/XOR-281)

**Result**

image

**Checklist**
none

**Testing instructions**
none

**Deployment instructions**
none

**Author concerns**
none

### CODE REVIEW

**Comment by squallstar** [comment]
@tonytlwu the way the UI has been laid out looks like I am sorting rules by status, whereas they are just sorted by order. I would have suggested the sorting happens on a new column called "Order", which shows a tooltip to the user explaining the reasons a user may want to sort rules.

This aside, the changes in the PR look fine from my point of view.

File: css/interface.css [hunk 1]
```diff
@@ -594,15 +594,19 @@ textarea.handsontableInput {
...
 
-#access-rules-list tr.disabled {
+#access-rules-list tr.rule-disabled a[data-toggle-status] .switch-label,
+#access-rules-list tr.rule-disabled > td {
   opacity: 0.5;
 }
 
+#access-rules-list tr.rule-disabled .opacity-full {
+  opacity: 1;
+}
```
Comment (tonytlwu) [inline css/interface.css:604]: Empty line between style declarations ([ref](https://docs.google.com/document/d/1XPlQHPbqp3rGGTekcC05bKtwKDDjrU2RMJ4VIHLXp64/edit#heading=h.q74tthq28fj3))

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Doesn't meet new coding standard

File: css/interface.css (hunk 1 above)
Comment (NikitaJadhavXoriant) [reply css/interface.css:604]: Empty line has been added as per new coding standards. Please review

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**